x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlQueryRequest.java

```diff
@@ -69,9 +69,7 @@ public EsqlQueryRequest(StreamInput in) throws IOException {
     public ActionRequestValidationException validate() {
         ActionRequestValidationException validationException = null;
         if (Strings.hasText(esqlVersion) == false) {
-            // TODO: make this required
-            // "https://github.com/elastic/elasticsearch/issues/104890"
-            // validationException = addValidationError(invalidVersion("is required"), validationException);
+            validationException = addValidationError(invalidVersion("is required"), validationException);
```

Comment:
I want to revert this before merging, so that the actual - breaking - API change becomes a single, small PR that can be easily reverted.

Reply:
Nevermind, discussed with @nik9000 : we can still revert just this in case something goes wrong.

Reply:
We're going to keep this in to save the few hours of build time. If something goes wrong we can back out just this line if we have to.